Sources/PrimerSDK/Classes/User Interface/Root/PrimerFormViewController.swift

```diff
@@ -30,12 +30,12 @@ class PrimerFormViewController: PrimerViewController {
                                      on stackView: UIStackView) {
         let theme: PrimerThemeProtocol = DependencyContainer.resolve()
 
-        let availablePaymentMethodsContainerStackView = UIStackView()
-        availablePaymentMethodsContainerStackView.translatesAutoresizingMaskIntoConstraints = false
-        availablePaymentMethodsContainerStackView.axis = .vertical
-        availablePaymentMethodsContainerStackView.alignment = .fill
-        availablePaymentMethodsContainerStackView.distribution = .fill
-        availablePaymentMethodsContainerStackView.spacing = 5.0
+        let availablePMethodsContainerStackView = UIStackView()
```

Comment:
Why did we go from `availablePaymentMethodsContainerStackView` to `availablePMethodsContainerStackView` here?

Reply:
Agree, the former naem was better

Reply:
Because the name was too long and Swiftlint was complaining. I will change it to `paymentMethodsStack`.